apps/typegpu-docs/src/components/ControlPanel.tsx

```diff
@@ -66,7 +66,7 @@ function SliderRow({
         min={min ?? 0}
         max={max ?? 1}
```

Comment:
We can also do the same here, giving initial values on destructuring.

```suggestion
        min={min}
        max={max}
```

Then give the initial value a default when using `useState`

```ts
const [value, setValue] = useState(initial ?? min);
```